mozilla-release/mobile/android/tests/cliqz/java/com/cliqz/mobile/test/CliqzRunner.java

```diff
@@ -0,0 +1,83 @@
+package com.cliqz.mobile.test;
+
+import android.util.Log;
+
+import org.junit.Ignore;
+import org.junit.internal.AssumptionViolatedException;
+import org.junit.internal.runners.model.EachTestNotifier;
+import org.junit.runner.Description;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runner.notification.StoppedByUserException;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.InitializationError;
+import org.junit.runners.model.Statement;
+
+public class CliqzRunner extends BlockJUnit4ClassRunner {
```

Comment:
Please add a `Copyright Cliqz` comment as documentation to the classes we define.

Reply:
done.